### PR TITLE
Add URL links to bread crumbs in `Suministro` detail (#61)

### PR DESCRIPTION
**Pull request by arielo**
Screen Shot 2020-01-15 at 3 53 57 PM

It definitely needs some styling but the links are working.

### CODE REVIEW

**Comment by gcollazo** [comment]
@froi pq está failing el action?

File: suministrospr/suministros/templates/suministros/suministro_detail.html [hunk 1]
```diff
@@ -12,9 +12,13 @@
   </div>
   <div class="flex justify-center">
     <div class="flex flex-row text-sm text-gray-600">
-      <div class="ml-2">{{ object.get_municipality_display }}</div>
+      <div class="ml-2">
+        <a href="{% url 'suministro-municipio-list' object.get_municipality_display|slugify %}">{{ object.get_municipality_display }}</a>
```
Comment (jpadilla) [inline suministrospr/suministros/templates/suministros/suministro_detail.html]:
```suggestion
        <a href="{% url 'suministro-municipio-list' object.municipality|slugify %}">{{ object.get_municipality_display }}</a>
```

**Comment by froi** [comment]
@gcollazo checking.

**Review by jpadilla — APPROVED** [review]
Gracias! 🎉

Left a minor comment/suggestion.

**Review by jpadilla — CHANGES_REQUESTED** [review]
@arielo Actually creo que necesitamos some classes for styling on those links...

End result should be similar to this https://suministrospr.com/sectores/frailes/

**Comment by froi** [comment]
@gcollazo seems to only happen when the PR is from a fork. May be a token scoping issue, I'm unsure. I'll look more into it.

That should stop us from merging/accepting things. It's not a action related to code or tests.

**Comment by arielo** [comment]
@jpadilla Only the `Suministro` model has a slug right now. Unless you meant to add a slug there as well.

**Comment by jpadilla** [comment]
> @jpadilla Only the `Suministro` model has a slug right now. Unless you meant to add a slug there as well.

@arielo 🤦‍♂ my bad, should be `object.municipality|slugify`

Update: I don't think we even need the `|slugify`  there 🤷‍♂

**Comment by arielo** [comment]
@jpadilla still not there. That might require some data model changes to since`municipality` is just string on the `suministro`

**Comment by froi** [comment]
Pay no attention to the failing action `greeting (pull_request)`. There is a scoping issue and I've just removed it from the workflow for `pull_requests`.

**Review by jpadilla — APPROVED** [review]

**Pull request merged**